crates/storage-query-datafusion/src/log/schema.rs

```diff
@@ -0,0 +1,51 @@
+// Copyright (c) 2023 - 2025 Restate Software, Inc., Restate GmbH.
+// All rights reserved.
+//
+// Use of this software is governed by the Business Source License
+// included in the LICENSE file.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0.
+
+#![allow(dead_code)]
+
+use datafusion::arrow::datatypes::DataType;
+
+use crate::table_macro::*;
+
+define_table!(
+    log(
+        /// Log ID
+        log_id: DataType::UInt32,
+
+        /// Segment index
+        segment_index: DataType::UInt32,
+
+        /// Segment start lsn
+        base_lsn: DataType::UInt64,
+
+        /// Segment provider kind
+        kind: DataType::Utf8,
+
+        // replicated loglet specific params
+
+        /// Loglet ID
+        loglet_id: DataType::UInt64,
```

Comment:
Can we use the display impl (string) of loglet ids in all those tables? It'd be consistent with how we present them in our log files and restatectl output.